UE4SS/src/LuaType/LuaUEnum.cpp

```diff
@@ -110,6 +110,240 @@ No overload found for function 'UEnum.ForEachName'.
             return 0;
         });
 
+        table.add_pair("GetEnumNameByIndex", [](const LuaMadeSimple::Lua& lua) -> int {
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.GetEnumNameByIndex'.
+Overloads:
+#1: GetNameByValue(integer Value))"};
+
+            auto& lua_object = lua.get_userdata<UEnum>();
+
+            if (!lua.is_integer())
+            {
+                lua.throw_error(error_overload_not_found);
+            }
+
+            auto value = lua.get_integer();
+            auto enum_pair = lua_object.get_remote_cpp_object()->GetEnumNameByIndex(value);
+
+            LuaType::FName::construct(lua, enum_pair.Key);
+            lua.set_integer(enum_pair.Value);
+
+            return 2;
+        });
+
+        table.add_pair("InsertIntoNames", [](const LuaMadeSimple::Lua& lua) -> int {
+            // Function: InsertIntoNames
+            // Param #1: Enum key name
+            // Param #2: Enum value
+            // Param #3: Enum index
+            // (Optional) Param #4: Shift values
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.InsertIntoNames'.
+Overloads:
+#1: InsertIntoNames(string Name, integer Value, integer Index) // shiftValues = true
```

Comment:
The default in UEnum.hpp is actually `false`, not `true`.
Why does the Lua binding deviate from this ?

Reply:
Double checked and you're right. Dunno how I missed that. Fixed.